rules/trackhub.smk

```diff
@@ -611,42 +611,41 @@ rule trackhub:
                 # ATAC-seq trackhub
                 if get_workflow() in ['atac_seq', 'chip_seq']:
                     for peak_caller in config['peak_caller']:
-                        for sample in breps[breps['assembly'] == assembly].index:
-                            bigpeak = f"{config['result_dir']}/{peak_caller}/{assembly}-{sample}.bigNarrowPeak"
-                            sample_name = f"{sample}{peak_caller}PEAK"
-                            sample_name = trackhub.helpers.sanitize(sample_name)
-
-                            if bigpeak:
-                                track = trackhub.Track(
-                                    name=sample_name,           # track names can't have any spaces or special chars.
-                                    source=bigpeak,             # filename to build this track from
-                                    visibility='dense',         # shows the full signal
-                                    tracktype='bigNarrowPeak',  # required when making a track
-                                    priority=priority
-                                )
-                                priority += 1
-                                trackdb.add_tracks(track)
-
-                        for sample in treps[treps['assembly'] == assembly].index:
-                            bigwig = f"{config['result_dir']}/{peak_caller}/{assembly}-{sample}.bw"
-                            assert os.path.exists(bigwig), bigwig + " not found!"
-                            sample_name = f"{sample}{peak_caller}BW"
+                        for brep in breps[breps['assembly'] == assembly].index:
+                            bigpeak = f"{config['result_dir']}/{peak_caller}/{assembly}-{brep}.bigNarrowPeak"
```

Comment:
what if you called your biological replicate the same as your technical replicate/sample?

this section of code can accept both, which may lead to ambiguity. the only way to get that ambiguity out (I think), is separating the outputs to different folders.

Reply:
The only thing I changed here is to make sure that the peak is plotted next the the bigwig track it belongs, not first all peaks, and then all bigwigs